tests/src/actionContainers/Swift3ActionContainerTests.scala

```diff
@@ -17,22 +17,29 @@
 package actionContainers
 
 import org.junit.runner.RunWith
-import org.scalatest.BeforeAndAfter
-import org.scalatest.FlatSpec
-import org.scalatest.Matchers
 import org.scalatest.junit.JUnitRunner
-import spray.json._
 
-import ActionContainer.withContainer
-
-import scala.util.Random
+import common.TestHelpers
+import common.TestUtils
+import common.Wsk
+import common.WskProps
+import common.WskTestHelpers
+import spray.json.DefaultJsonProtocol.StringJsonFormat
+import spray.json.JsObject
+import spray.json.JsString
+import spray.json.pimpAny
 
 @RunWith(classOf[JUnitRunner])
-class Swift3ActionContainerTests extends SwiftActionContainerTests {
+class Swift3ActionContainerTests extends SwiftActionContainerTests
+    with TestHelpers
+    with WskTestHelpers {
 
     override val checkStdOutEmpty = false
     override val swiftContainerImageName = "whisk/swift3action"
 
+    implicit val wskprops = WskProps()
+    val wsk = new Wsk()
```

Comment:
The tests below are good, but they should be in a separate file; the `*ContainersTests` are supposed to be runnable without a Whisk deployment (i.e. you only need Docker and the appropriate images).